components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -439,6 +439,14 @@ - (void)viewDidDisappear:(BOOL)animated {
   self.headerShadowLayer = nil;
 }
 
+- (void)preferredContentSizeDidChangeForChildContentContainer:(id<UIContentContainer>)container {
+  [super preferredContentSizeDidChangeForChildContentContainer:container];
+  _contentHeaderTopInset = NSNotFound;
+  _contentHeightSurplus = NSNotFound;
+  _addedContentHeight = NSNotFound;
```

Comment:
This is a peculiar use of NSNotFound. Did you consider using a bool to indicate whether or not these values are valid instead?

NSNotFound docs:
> A value indicating that a requested item couldn’t be found or doesn’t exist.
>
> NSNotFound is typically used by various methods and functions that search for items in serial data and return indices, such as characters in a string object or id objects in an NSArray object.

Reply:
I agree that this use isn't optimal, but to change that mechanism I'll have to do a big cleanup as the whole code relies on NSNotFound for its properties. This is outside the scope of this PR. I've opened https://github.com/material-components/material-components-ios/issues/5635 for tracking.